features/step_definitions/game_steps.rb

```diff
@@ -121,4 +121,9 @@
   steps %Q{
     And I press "Submit New Game"
   }
-end
\ No newline at end of file
+end
+
+When /^the game "()" should expire in one day$/ do |game|
+  game = Game.where(:title => game)[0]
+  expectgame.expiration
```

Comment:
Well I deleted it with the inline editor, and it doesn't show up in github or my local copy anymore except for in this review. so that's fixed